Altis_Life.Altis/stringtable.xml

```diff
@@ -4694,7 +4694,7 @@
             <Spanish>Por ser arrestado se te han quitado estas licencias (si las tienes)\n\nL. de Armas \nL. Rebelde</Spanish>
             <Russian></Russian>
             <German>Bei der Verhaftung hast du folgende Lizenzen verloren, sofern du sie besessen hast:\n\nWaffenschein\nRebellenausbildung.</German>
-            <French>Après votre mise en prison, vous avez perdu les licences suivante si vous les aviez\n\nPermis de port d'arme\n LicenceRebelle</French>
+            <French>Après emprisonnement, vous avez perdu les licences suivante si vous les aviez\n\nPermis de port d'arme\n LicenceRebelle</French>
```

Comment:
"Après **votre** emprisonnement"

Reply:
"Après votre emprisonnement" or "Après emprisonnement" : The both are correct.

Reply:
"Après emprisonnement" is more implicit. Adding "votre" makes the sentence better to say / read.

Reply:
Plural : "licences suivante" => "licences suivantes"